### PR TITLE
313 payment for spent gas at the end of each iteration

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**
Added account metas for neon-evm instructions.
Fixed data structure layouts

### CODE REVIEW

File: proxy/plugin/solana_rest_api_tools.py [hunk 1]
```diff
@@ -68,7 +68,6 @@
     "trx_count" / Bytes(8),
     "code_account" / Bytes(32),
     "ro_blocked_cnt" / Int8ul,
-    "is_rw_blocked_acc" / Int8ul,
     "rw_blocked_acc" / Bytes(32),
     "eth_token_account" / Bytes(32),
 )
```
Comment (otselnik) [inline proxy/plugin/solana_rest_api_tools.py]: You need to pull changes from develop branch. There is a correct version of `ACCOUNT_INFO_LAYOUT`

File: proxy/plugin/solana_rest_api_tools.py [hunk 2]
```diff
@@ -68,7 +68,7 @@
     "nonce" / Int8ul,
     "trx_count" / Bytes(8),
     "code_account" / Bytes(32),
-    "is_rw_blocked" / Int8ul,
+    "ro_blocked_cnt" / Int8ul,
```
Comment (otselnik) [inline proxy/plugin/solana_rest_api_tools.py]: Rename to `is_rw_blocked` as in develop branch.

**Review by otselnik — DISMISSED** [review]

File: proxy/plugin/solana_rest_api_tools.py (hunk 1 above)
Comment (vasiliy-zaznobin) [reply proxy/plugin/solana_rest_api_tools.py]: Fixed

File: proxy/plugin/solana_rest_api_tools.py (hunk 2 above)
Comment (vasiliy-zaznobin) [reply proxy/plugin/solana_rest_api_tools.py]: Fixed

File: proxy/plugin/solana_rest_api_tools.py [hunk 3]
```diff
@@ -68,7 +68,6 @@
     "nonce" / Int8ul,
     "trx_count" / Bytes(8),
     "code_account" / Bytes(32),
-    "is_rw_blocked" / Int8ul,
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api_tools.py]: Restore this field.

Comment (vasiliy-zaznobin) [reply proxy/plugin/solana_rest_api_tools.py]: Fixed

**Review by s-medvedev — DISMISSED** [review]

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**